src/components/ActionsMenu/ActionsMenu.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import styles from './ActionsMenu.st.css';
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+import { ActionsMenuItem } from './Item/ActionsMenuItem';
+import { ActionsMenuDivider } from './Divider/Divider';
```

Comment:
You should import from `./Divider`
This way we can change the internal folder structure more easily